libs/src/sdk/api/albums/types.ts

```diff
@@ -0,0 +1,155 @@
+import { z } from 'zod'
+import type { CrossPlatformFile as File } from '../../types/File'
+import { Genre } from '../../types/Genre'
+import { HashId } from '../../types/HashId'
+import { Mood } from '../../types/Mood'
+import { isFileValid } from '../../utils/file'
+import { createUploadTrackMetadataSchema } from '../tracks/types'
+
+export const createUploadAlbumMetadataSchema = () =>
+  z
+    .object({
+      description: z.optional(z.string().max(1000)),
+      genre: z.enum(Object.values(Genre) as [Genre, ...Genre[]]),
```

Comment:
We can remove some of these fields from the upload/create playlist schema now, bc playlists don't support genre, isrc, iswc, license, etc

Reply:
Actually I think we might accept some/all of these fields for `uploadPlaylist` based on our UI (even if we don't use them anywhere). That being said, I think it'd be fine to exclude them (except maybe Mood and Genre) for simplicity.

<img width="1185" alt="Screenshot 2023-07-20 at 7 46 07 PM" src="https://github.com/AudiusProject/audius-protocol/assets/36916764/5e5db26d-5b35-4521-827b-7b377f7aea73">